### PR TITLE
fix: automation API update fix

### PR DESCRIPTION
**Pull request by rileymd88**
Small bug fix to handle new automations API

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Hi, and thanks for contributing!

Note: If you are part of Qlik R&D, you can apply to the `open-source` team in our _internal_ GitHub organization, and ignore the rest of these instructions.

Before we can handle your pull request we need you to sign our contributor license agreement.
This is a one-time process and enables you to contribute without hassle to any of the qlik-oss repositories in the future.

And it only takes a few minutes! Here are some useful links to get you started:

* [Sign the CLA](https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sign-the-cla)
* [Rationale behind the CLA](https://github.com/qlik-oss/open-source/blob/master/cla-rationale.md)
* [Qlik R&D Open Source repository](https://github.com/qlik-oss/open-source#open-source-at-qlik-rd)
* [Official Slack team](http://branch.qlik.com/slack)

Having troubles? Feel free to reach out to us either here or on Slack!

**Note:** Once the CLA is sorted, you should rebase/touch this pull request to trigger an update of this CLA status.

**Review by niekvanstaveren — COMMENTED** [review]
Please make sure to sign up to the open source team

File: src/utils/actions.js [hunk 1]
```diff
@@ -298,9 +298,9 @@ const actions = [
             const autoInfo = await fetch(`../api/v1/automations/${itemInfo.resourceId}`).then((response) =>
               response.json()
             );
-            let executePath = `../api/v1/automations/${autoInfo.guid}/actions/execute?X-Execution-Token=${autoInfo.execution_token}`;
+            let executePath = `../api/v1/automations/${autoInfo.id}/actions/execute?X-Execution-Token=${autoInfo.executionToken}`;
             if (automationPostData) {
-              const inputBlocks = await fetch(`../api/v1/automations/${itemInfo.resourceId}/blocks`)
+              const inputBlocks = autoInfo.workspace.blocks.find(block => block.type === 'FormBlock').form ?? []
```
[the remark this answers is not in the document]
Comment (rileymd88) [reply src/utils/actions.js:303]: Yup you are correct I took this code from the other pull request - if we leave this line the same then I think it should work fine

[the remark this answers is not in the document]
Comment (rileymd88) [reply src/utils/actions.js]: I added a fix which should have the input blocks working again with the new automation API changes

**Review by niekvanstaveren — APPROVED** [review]

**Pull request merged**